scripts/lockfile/lockfile.go

```diff
@@ -20,7 +20,9 @@
 
 package main
 
-// This .go file is used to test the lockfile package (m3/src/x/lockfile)
+// This .go file is used to test the lockfile implementation in dbnode/server.
+// The implementation is internal to dbnode/server, but is utilized by this script
+// so that its test can exec new processes that acquire file locks.
```

Comment:
lockfile_test.go execs this script to intentionally spin up a new process. The script accordingly runs in a `main` package, meaning we're not able to make the APIs internal